### PR TITLE
fix: Refresh history component after actions user has made

### LINKED ISSUE #1290: Refresh history component after actions user has made
When I do listing on my own item, it leaves me open List input after submitting transaction.

User: What's happening? Was it listed?

We should hide it once tx was submitted to not mislead people list again accidentally

image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1290
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/rmrk/Gallery/AvailableActions.vue [hunk 1]
```diff
@@ -386,6 +386,7 @@ export default class AvailableActions extends mixins(
               `[${this.selectedAction}] ${this.nftId}`,
               notificationTypes.success
             )
+            this.$emit('change', this.selectedAction)
```
Comment (kkukelka) [inline components/rmrk/Gallery/AvailableActions.vue:389]: hmm why do we need to emit `this.selectedAction` here though? you are not doing anything with the function parameter in parent

**Review by kkukelka — CHANGES_REQUESTED** [review]

**Review by kkukelka — APPROVED** [review]
all good now

**Comment by kkukelka** [comment]
pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 150.25 USD/KSM ~ 0.333 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xdb90821b5a7a8c7c3d09ad2f8ff8f00038c20cf84f273f5161ca1d509695494e](https://kusama.subscan.io/extrinsic/0xdb90821b5a7a8c7c3d09ad2f8ff8f00038c20cf84f273f5161ca1d509695494e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by kkukelka** [comment]
cant merge yet . hmmhmhm. i assume because of edge release destroying build process

**Comment by yangwao** [comment]
Seems broken by a recent merge? 

@Jarsen136 


https://user-images.githubusercontent.com/5887929/163154827-d453c4ac-da0a-4556-82c9-11e1f27ca7f3.mov

**Comment by Jarsen136** [comment]
> Seems broken by a recent merge? 
> 
> 
> 
> @Jarsen136 
> 
> 
> https://user-images.githubusercontent.com/5887929/163154827-d453c4ac-da0a-4556-82c9-11e1f27ca7f3.mov
> 
> 
> 
> 
I will check it later.
This pr has not merged into main branch

**Comment by Jarsen136** [comment]
Seems that I could not reproduce this bug.
Could you pls have a try again ❓   @yangwao

**Comment by yangwao** [comment]
yeah can't reproduce either lol, even tried cache and it happened so.. dunno! 
just someone could test this i.e. like buying something for example :)

**Comment by yangwao** [comment]
tried here and haven't refreshed history :| 

but I clicked off loader.. maybe that's why?

image

**Comment by yangwao** [comment]
yeah it did not work even a second time. :| 
How did you test it? 

image

**Comment by yangwao** [comment]
oh I was doing buy! so let me try LIST

**Comment by Jarsen136** [comment]
Sorry, It may be because the data fetched immediately has not been updated after the `list、buy` action is submitted.  : )

I make history events to polling to ensure that events can update when they change.

**Comment by Jarsen136** [comment]
WFM just now, by doing buy action and list action on-chain.

**Comment by yangwao** [comment]
wfm 

I just noticed that on collection tile switcher (large display small display) went missing?
image


https://kodadot.xyz/rmrk/collection/d6e71489d50f818c09-TEWIN
image

**Comment by yangwao** [comment]
pay 100 usd

**Pull request merged**